index.js

```diff
@@ -45,19 +47,26 @@ module.exports = function (content) {
         .replace(new RegExp(escapeRegExp(pathSep) + '+', 'g'), pathSep);
     var html;
 
-    if (content.match(/^module\.exports/)) {
+    if (content.match(/(?:^module\.exports)|(?:^export\s+default)|(?:^exports.default)/)) {
         var firstQuote = findQuote(content, false);
         var secondQuote = findQuote(content, true);
         html = content.substr(firstQuote, secondQuote - firstQuote + 1);
     } else {
         html = content;
     }
 
+    var exportsString = "module.exports = ";
```

Comment:
Works for me!
OTOH it feels more natural to me to specify the export format at that loader whose output is actually consumed by the app.
Also that would not work with raw-loader, because that loader is hard coded to commonjs exports.
I don't know how important that scenario is, because I am new at webpack. Are there valid reasons for chossing the raw-loader over the html-loader?